requirements/requirements-optionals.txt

```diff
@@ -2,3 +2,4 @@
 markdown==2.5.2
 django-guardian==1.3.2
 django-filter==0.10.0
```

Comment:
I was on the fence with that possibility. My thought was that, given Django's strong recommendation to install pytz, this would be a potentially acceptable "failure". The Django docs actually state:

> Installing pytz is highly recommended, but may not be mandatory depending on your particular database backend, operating system and time zone. If you encounter an exception querying dates or times, please try installing it before filing a bug.

Also, again, django-timezone-field is the primary case I know of (although there obviously could be others) where tzinfo objects would be output for serialization, and that does explicitly require pytz.